src/main.cpp

```diff
@@ -300,7 +300,7 @@ void setMotorSpeeds(Speeds requestedMotorSpeeds, Servo &motorLeft, Servo &motorR
     // for autonomous control we could revert back to using full scale
     // but for manual control, and for testing speedcontrol precision
     // better to start with limiting to lower speeds
-    float maxspeed_mm_per_sec = 3000;  //max acheivable is 8000
+    float maxspeed_mm_per_sec = 1000;  //max acheivable is 8000
```

Comment:
I think this comment might need updating?

Reply:
good catch. I don't know what the new max speed is, but in theory its closer to 3000mm/s